src/main/kotlin/com/exactpro/th2/http/client/util/MessageUtil.kt

```diff
@@ -100,16 +101,11 @@ private fun createRequest(head: Message, body: RawMessage): RawHttpRequest {
     val parentEventId = head.parentEventId.id.ifEmpty { body.parentEventId.id }
     val metadataProperties = body.metadata.propertiesMap + head.metadata.propertiesMap
 
-    head.metadata.propertiesMap.forEach{it ->
-        val name = it.key
-        val value = it.value
-
-        if (!(name.isNullOrEmpty() || value.isNullOrEmpty())) {
-            if (name.startsWith("header-", ignoreCase = true)) {
-                val final = name.substring(7)
+    head.metadata.propertiesMap.forEach{ (name, value) ->
+            if (name.startsWith(HEADER_PREFIX, ignoreCase = true)) {
+                val final = name.substring(HEADER_PREFIX.length)
```

Comment:
Can be inlined

Reply:
do you mean like this 
`head.metadata.propertiesMap.forEach { (name, value) ->
        if (name.startsWith(HEADER_PREFIX, ignoreCase = true))
            httpHeaders.with(name.substring(HEADER_PREFIX.length), value)}`

Reply:
Yeah, but with `forEach`'s body on a separate line

Reply:
I meant smth like this:
```
metadataProperties.forEach{ (name, value) ->
    if (name.startsWith(HEADER_PREFIX, ignoreCase = true)) {
        httpHeaders.with(name.substring(HEADER_PREFIX.length), value)
    }
}
```

Reply:
I initially wrote it that way, it's just the comment changed the layout

Reply:
Use triple ` to keep newlines